lib/rubocop/cop/lint/literal_as_condition.rb

```diff
@@ -98,13 +99,43 @@ def message(node)
 
         def check_for_literal(node)
           cond = condition(node)
-          if cond.literal?
-            add_offense(cond)
+          if cond.truthy_literal?
+            autocorrect_redundant_truthy_condition(node, cond)
+          elsif cond.falsey_literal?
+            autocorrect_redundant_falsey_condition(node, cond)
```

Comment:
Can you add test cases for handling falsey literals?

Reply:
Added.